data-remote/src/main/java/com/michaldrabik/data_remote/tmdb/api/TmdbApi.kt

```diff
@@ -21,10 +21,9 @@ internal class TmdbApi(private val service: TmdbService) : TmdbRemoteDataSource
       if (showTmdbId == null || showTmdbId <= 0) TmdbImages.EMPTY
       if (season == null || season <= 0) TmdbImages.EMPTY
       if (episode == null || episode <= 0) TmdbImages.EMPTY
-      val images = service.fetchEpisodeImages(showTmdbId, season, episode)
-      images.stills?.firstOrNull()
```

Comment:
It can now be reverted to return a single image or null as it used to be. All callers of this method work with just a single still in the end.